### ISSUE TITLE
Update BDD rules with multi language support 

### ISSUE THREAD

**Issue by bhirsz** [MarketSquare/robotframework-robocop#647]
Related #646

**Comment by mnojek** [comment]
@bhirsz Can you explain what should be done here?

**Comment by bhirsz** [comment]
We have some rules regarding BDD syntax. It only supports english names, and we would need to support translated keywords like "Jeżeli" "Und"..

**Comment by mnojek** [comment]
Are they all available through RF API?

**Comment by bhirsz** [comment]
No.. its bit more complicated with BDD. We need to load the language - user need to provide langage shebang or use ``--langauge`` option . Then we can use robot API to load the possible BDD keywords and use for rules. I did it on the translate transformer in Robotidy: https://github.com/MarketSquare/robotframework-tidy/blob/main/robotidy/transformers/Translate.py

If we have some rules that rely on keyword name, we would also need to give BDD support for it (ie Then Return From Keyword?).

**Comment by bhirsz** [comment]
Marked as won't do. There are no requests for this, if there are any users that use translated BDD syntax, they should open new issues and we will analyze whether to support this.
